src/frontend/src/test_utils.rs

```diff
@@ -310,6 +338,13 @@ impl MockCatalogWriter {
         Ok(source.id)
     }
 
+    fn create_sink_inner(&self, mut sink: ProstSink) -> Result<u32> {
```

Comment:
```suggestion
    fn create_sink_inner(&self, mut sink: ProstSink) -> Result<()> {
```

Seriously, if we don't use the id, why do we return it?